ai-platform/snippets/predict-text-embeddings.js

```diff
@@ -44,11 +44,12 @@ async function main(
     const [response] = await client.predict(request);
     console.log('Got predict response');
     const predictions = response.predictions;
-    for (const prediction of predictions) {
-      const embeddings = prediction.structValue.fields.embeddings;
-      const values = embeddings.structValue.fields.values.listValue.values;
-      console.log('Got prediction: ' + JSON.stringify(values));
-    }
+    const embeddings = predictions.map(p =>
+      p.structValue.fields.embeddings.structValue.fields.values.listValue.values.map(
```

Comment:
Can this be split up to make the line and statement shorter?
```js
p => {
  const embeddingsProto = p.structValue.fields.embeddings
  const valuesProto = structValue.fields.values
  return valuesProto.listValue.values.map(v => v.numberValue)
}
```

(Not sure if they're protos or some other value representation)

Reply:
They are embeddingsStructProto and valuesListProto.